.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+on: [push, pull_request, workflow_call, workflow_dispatch]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set CACHE_DATE
+      run: eval echo 'CACHE_DATE="$(date --utc +%Y%m%dT%H%M%SZ)"' >> $GITHUB_ENV
+    - run: ./devel/pull
+    - run: ./devel/build
+    - if: ${{ github.event_name == 'push' }}
+      name: Docker login
+      run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u ${{ secrets.DOCKER_LOGIN }} --password-stdin
```

Comment:
For this, I created a new Access Token with description "docker-base github actions" on the nextstrainbot Docker user.

Reply:
:+1: for the new access token. Let's use the `docker/login-action` here instead of calling `docker login`? 

(Will review more later.)

Reply:
Done.